js/base.js

```diff
@@ -26,7 +26,7 @@ export class API extends mix(BaseAPI).with(LoadAPI, RenderAPI, SceneAPI) {
         const auth = options.kwargs.auth === undefined ? true : options.kwargs.auth;
         delete options.kwargs.auth;
         if (auth) {

```

Comment:
Couldn't the token check be made here as well?

Reply:
Just keeping consistency as I saw this `if (auth)` separated everywhere and based of the [example](https://github.com/ripe-tech/ripe-twitch-api-js/blob/75681636c03253f667810bf3a95e4c8e531a9f7f/js/base.js#L45) 

If you don't think it makes sense for this case, I'll add the condition to that. What do you say?

Reply:
Seems weird having an if inside another if and no other code. I would say join the two ifs